sections/main-collection-banner.liquid

```diff
@@ -1,5 +1,15 @@
 {{ 'component-collection-hero.css' | asset_url | stylesheet_tag }}
 
+{%- if settings.media_shadow_vertical_offset > 0 -%}
+  {%- style -%}
+    @media screen and (max-width: 749px) {
+      .collection-hero--with-image .collection-hero__inner {
+        padding-bottom: calc({{ settings.media_shadow_vertical_offset }}px + 2rem);
+      }
+    }
+  {%- endstyle -%}
+{%- endif -%}
```

Comment:
I'm wondering if we need the `if` here.  If we still want there to be a minimum padding of `2rem`, then it would just add `0 + 2rem` and still end up as `2rem`.

Reply:
But since the vertical offset can be negative then I think we end up with a negative padding value 🤔

Reply:
But because of negative values of the vertical offset, it would mess with the padding and end up being `0`. Seems like padding doesn't work with negative value so it will default to `0` 🤔

Reply:
Good point - we could use `at_least: 0`.

https://shopify.dev/api/liquid/filters/math-filters

Reply:
Ah yes, that's great 👌  I made the update.

Reply:
Ah smart idea! I made the change 👌